components/ImageR/imageR.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="imageR.css" />
+    <link rel="stylesheet" href="/css/styles.css" />
```

Comment:
Always keep your stylesheets at the bottom of the head to make sure that your styles get applied to the element

Reply:
okay